packages/dd-trace/src/telemetry/send-data.js

```diff
@@ -1,4 +1,26 @@
 const request = require('../exporters/common/request')
+const { isTrue } = require('../util')
+
+const debug = process.env.DD_TELEMETRY_DEBUG
```

Comment:
this should be done in config.js instead no ?

Reply:
It is a internal flag to enable debugging requests, that's the reason I left it here. But I can move it to config.js.

Ah, [telemetry/index.js](https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/telemetry/index.js#L9) reads `DD_TELEMETRY_HEARTBEAT_INTERVAL` property in the same way (in the file itself instead of in config.js)...

Reply:
Ah, well idk then, ask the tracer team

Reply:
I could move to config.js both...

Reply:
moved both!